infinispan-client/src/test/java/io/quarkus/ts/openshift/infinispan/client/InfinispanGreetingResourceOpenShiftIT.java

```diff
@@ -83,6 +92,25 @@ public void testHelloSecondEndpoint() {
                 .body(is("Hello World, Infinispan is up and changed!"));
     }
 
+    // The second test scenario - kill Quarkus pod
+    @Test
+    @Order(3)
+    public void testSimpleFailOver() {
+        openshift.rolloutChanges(metadata.appName);
+
+        when()
+                .get("/first-endpoint")
+                .then()
+                .statusCode(200)
+                .body(is("Hello World, Infinispan is up and changed!"));
+
+        when()
+                .get("/second-endpoint")
+                .then()
+                .statusCode(200)
+                .body(is("Hello World, Infinispan is up and changed!"));
```

Comment:
...and here, we'd assume the number of requests on counter is what we expect it to be :)